lib/route_translator/extensions/route_set.rb

```diff
@@ -5,12 +5,42 @@
 module ActionDispatch
   module Routing
     class RouteSet
+      module VerificationLambdas
```

Comment:
can we move this to another file, like `lib/route_translator/translator/path.rb`?